htdocs/xoops_lib/modules/protector/library/HTMLPurifier/AttrTransform/TargetNoreferrer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+// must be called POST validation
+
+/**
+ * Adds rel="noreferrer" to any links which target a different window
+ * than the current one.  This is used to prevent malicious websites
+ * from silently replacing the original window, which could be used
+ * to do phishing.
+ * This transform is controlled by %HTML.TargetNoreferrer.
+ */
+class HTMLPurifier_AttrTransform_TargetNoreferrer extends HTMLPurifier_AttrTransform
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)
* Class name "HTMLPurifier_AttrTransform_TargetNoreferrer" is not in camel caps format

<!-- Comment by [SideCI](https://sideci.com) -->